.github/workflows/acceptance.yml

```diff
@@ -276,9 +276,9 @@ jobs:
         PYTEST_ADDOPTS: "--reruns 5"
 
         # services configuration
-        SENTRY_KAFKA_HOSTS: kafka:9093
-        SENTRY_ZOOKEEPER_HOSTS: zookeeper:2182
-        SENTRY_REDIS_HOST: redis
+        SENTRY_KAFKA_HOSTS: 127.0.0.1:9092
+        SENTRY_ZOOKEEPER_HOSTS: sentry_zookeeper:2181
+        SENTRY_REDIS_HOST: sentry_redis
```

Comment:
```suggestion
```

This isn't needed, I don't think.

Reply:
I think we can remove this across the board.

Reply:
Yeah I think you're right, since this has been wrong